docs/csharp/language-reference/keywords/index.md

```diff
@@ -115,7 +115,7 @@ A contextual keyword is used to provide a specific meaning in the code, but it i
         [and](../operators/patterns.md#logical-patterns)  
         [alias](extern-alias.md)  
         [ascending](ascending.md)  
-        [args](../../fundamentals/program-structure/top-level-statements.md#args)
+        [args](../../fundamentals/program-structure/top-level-statements.md#args)  
```

Comment:
You can also use a backslash, which won't be mistakenly trimmed off. Just FYI though.

```suggestion
        [args](../../fundamentals/program-structure/top-level-statements.md#args)\
```